jsons/deserializers/default_date.py

```diff
@@ -0,0 +1,20 @@
+from datetime import date
+import re
+
+from jsons._date_impl import get_date_inst, DATE_PARSE_PATTERN
+
+
+def default_date_deserializer(obj: str,
+                                  cls: type = date,
```

Comment:
Forgive me for being a style-nazi here, but you indented with 4 spaces too many. 🙂